querybook/server/logic/metastore.py

```diff
@@ -393,6 +398,33 @@ def create_table_ownership(table_id, uid, commit=True, session=None):
     return table_ownership
 
 
+@with_session
+def create_table_ownerships(
+    table_id: int, owners: list[DataOwner] = [], commit=True, session=None
+):
+    """This function is used for loading owners from metastore."""
+    # delete all the ownerships of the table first
+    session.query(DataTableOwnership).filter_by(data_table_id=table_id).delete()
+
+    for owner in owners:
+        user = get_user_by_name(owner.username)
+        if not user:
+            LOG.error(
```

Comment:
gotta think of internal cases where the name is invalid

Reply:
If the name is invalid, then it can't be found, this `if not uesr` will catch it. or are you talking about even the name is invalid, we should still show it?